test/check-application

```diff
@@ -1030,14 +1055,30 @@ class TestFiles(testlib.MachineCase):
 
         self.enter_files()
 
+        create_dirs = "mkdir -p"
+        for dirname in ["testdir", "anotherdir"]:
+            create_dirs += f" /home/admin/{dirname}"
+        m.execute(create_dirs)
```

Comment:
Feels a bit overengineered for:
m.execute("""
mkdir /home/admin/testdir
mkdir /home/admin/anotherdir
""")

Also these will be owned by root.